Framework/DataHandling/src/LoadFITS.cpp

```diff
@@ -436,17 +436,15 @@ void LoadFITS::doLoadFiles(const std::vector<std::string> &paths, const std::str
   size_t fileNumberInGroup = 0;
   WorkspaceGroup_sptr wsGroup;
 
-  if (!AnalysisDataService::Instance().doesExist(outWSName)) {
-    wsGroup = std::make_shared<WorkspaceGroup>();
-    wsGroup->setTitle(outWSName);
-  } else {
+  if (AnalysisDataService::Instance().doesExist(outWSName)) {
```

Comment:
Use if initialisers for this rather than getting the ADS instance multiple times:
```c++
if (auto &ads = AnalysisDataService::Instance(); ads.doesExist(outWsName)){
    wsGroup = ads.retrieveWS<WorkspaceGroup>(outWSName);
```

Reply:
done